packages/app-desktop/gui/ChatPanel/style.scss

```
@@ -0,0 +1,202 @@
+	font-style: italic;
+	text-align: center;
+	margin-top: 20px;
+	padding: 0 20px;
+	line-height: 1.5;
+}
+
+.chat-panel > .messages > .turn {
+	padding: 8px 10px;
+	border-radius: 6px;
+	white-space: pre-wrap;
+	word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)